spec/unit/try_spec.rb

```diff
@@ -57,6 +57,10 @@
     it { is_expected.to eql(described_class.new([ZeroDivisionError], "foo")) }
     it { is_expected.not_to eql(error[division_error]) }
 
+    it "deconstructs to the value" do
+      expect(subject.deconstruct).to eq ["foo"]
+    end
```

Comment:
can you please test it with pattern matching rather than hitting `.deconstruct` directly? It's done like this in other tests. Also equality is checked via `eql` rather than `eq`

Reply:
Thanks for your review @flash-gordon - please see 9caca77